packages/effect/src/Stream.ts

```diff
@@ -4566,6 +4566,10 @@ export const someOrFail: {
  * @category utils
  */
 export const split: {
+  <A, B extends A>(
+    refinement: Refinement<A, B>
+  ): <E, R>(self: Stream<A, E, R>) => Stream<Chunk.Chunk<Exclude<A, B>>, E, R>
+  <A, E, R, B extends A>(self: Stream<A, E, R>, refinement: Refinement<A, B>): Stream<Chunk.Chunk<Exclude<A, B>>, E, R>
```

Comment:
move this signature down

(for code consistency you can see for example `Stream.takeWhile`)